sipa/static/js/agdsn.js

```diff
@@ -0,0 +1,47 @@
+function guess_locale() {
```

Comment:
The locale detection implemented in the backend should be used, searching all cookies is not good solution.

The backend's decision could be made available by a separate JSON endpoint or just be stored as inline JavaScript by the template.

Reply:
The inline JS solution sounds like a hack too :/ (even though the result will be better than the current locale guessing).

The JSON endpoint sounds good. I'm just wondering if it has the potential to leak any data about the user to third parties.

Reply:
I have included a JS function in the header of base.html that returns the current language defined by the backend, so every page can use it (e.g. authenticated contact later on). (3bc111b)

Reply:
Please see my comment on 3bc111bbb814457fa39764b809e5879fe3ffbaea.